tdvt/tdvt/exprtests/initialsql/setup.initialsql.notemptable.str.txt

```diff
@@ -0,0 +1 @@
+RAWSQLAGG_STR("select str1 from Calcs where str1 = 'BINDER CLIPS'")
```

Comment:
If temp table is not referenced, what are we testing here? The setup files seems unnecessary to me.